docs/content/ingestion/firehose.md

```diff
@@ -144,7 +142,10 @@ A sample ingest Firehose spec is shown below:
 
 ### SqlFirehose
 
-This Firehose can be used to ingest events residing in RDBMS. The database connection information is provided as part of the ingestion spec. For each query, the results are fetched locally and indexed. If there are multiple queries from which data needs to be indexed, queries are prefetched in the background upto `maxFetchCapacityBytes` bytes.
+This Firehose can be used to ingest events residing in an RDBMS. The database connection information is provided as part of the ingestion spec.
+For each query, the results are fetched locally and indexed.
+If there are multiple queries from which data needs to be indexed, queries are prefetched in the background, up to `maxFetchCapacityBytes` bytes.
+This firehose works with `map` typed parsers.
```

Comment:
Could you include an example of what the parser should look like? I think this language here isn't enough to make it clear what people need to do. It is better (before, it was literally impossible for someone to know what to do just by reading the docs) but still deserves an example due to the distance between where parsers and firehoses live in the ingestion specs.

Reply:
Btw, when I say example I mean something like this:

```
      "parser": {
        "type" : "map",
        "parseSpec": {
          "format": "timeAndDims",
          "dimensionsSpec": {
            "dimensions": [
              "dim1",
              "dim2",
              "dim3"
            ]
          },
          "timestampSpec": {
            "format": "auto",
            "column": "ts"
          }
        }
      }
```

(Posted in the Apache Druid Slack channel yesterday)

Reply:
I added the examples to the respective sql extension pages since they did not mention they implemented the sql firehose, and suggested that more detailed examples can be found there